src/problems/basic_problems.jl

```diff
@@ -299,15 +299,17 @@ Any extra keyword arguments are captured to be sent to the optimizers.
 
 ### Fields
 
-* `f`: The function in the problem.
-* `u0`: The initial guess for the optima.
-* `p`: The parameters for the problem. Defaults to `NullParameters`.
+* `f`: the function in the problem.
+* `u0`: the initial guess for the optima.
+* `p`: the parameters for the problem. Defaults to `NullParameters`.
 * `lb`: the lower bounds for the optimization of `u`.
 * `ub`: the upper bounds for the optimization of `u`.
-* `lcons`:
-* `ucons`:
-* `sense`:
-* `kwargs`: The keyword arguments passed on to the solvers.
+* `lcons`: the vector of lower bounds for the constraints passed to [OptimizationFunction](@ref).
+    Defaults to `nothing`, implying no lower bounds for the constraints.
+* `ucons`: the vector of upper bounds for the constraints passed to [OptimizationFunction](@ref).
+    Defaults to `nothing`, implying no upper bounds for the constraints.
+* `sense`: the objective sense, can take `MaxSense` or `MinSense` from Optimization.jl.
```

Comment:
These should be defined in SciMLBase, otherwise you cannot fully define the problem without the solver interface 😅.

Reply:
Also, is it actually implemented everywhere? I don't think it's well tested? I'm scared to say it exists if it's ignored and doesn't throw errors on non-compliant solvers.

Reply:
> otherwise you cannot fully define the problem without the solver interface

True, I'll move it here

>is it actually implemented everywhere? I don't think it's well tested?

Not everywhere - https://github.com/SciML/Optimization.jl/search?q=MAXSENSE but it is tested

>I'm scared to say it exists if it's ignored and doesn't throw errors on non-compliant solvers.

It doesn't throw errors that's true, but it should be easy to add and I think it is a part of the larger issue https://github.com/SciML/Optimization.jl/issues/282, I'll make a note of it there, should be handled together